templates/shared/forms/form-fields.html

```diff
@@ -0,0 +1,208 @@
+<section class="p-section">
+  <form {% if isModal %}class="js-modal-form"{% endif %}
+        action="/marketo/submit"
```

Comment:
```suggestion
        action="https://ubuntu.com/marketo/submit"
```
This needs to be sent to u.com not through c.com as we don't have a local api

Reply:
Thanks for helping here